app.js

```diff
@@ -39,28 +43,48 @@ mongoose.connect(`mongodb://mongo_db:27017/${db.name}`,
     console.log(db)
     console.log('MongoDB Error:', err)
 })
+
+
 app.set('view engine', 'ejs')
 
 
 //Redis setup
 const redisClient = redis.createClient({host: 'redis'}) //Uses default PORT: 6379
 
+//DDoS prevention
+app.use((req, res, next) => {
+    if(tooBusy()) {
+        return res.sendStatus(503)
+    }
+    next()
+})
+
+//Bruteforce prevention
+const apiLimiter = rateLimit({
+    windowMs: 15 * 60 * 1000, //15 minutes
```

Comment:
One request per 15 minutes?

Reply:
https://www.npmjs.com/package/express-rate-limit

I've set it to 100 requests per 15 minutes